tests/gsrpc/extrinsics/test_bal/availaccountbalance.go

```diff
@@ -0,0 +1,299 @@
+package main
```

Comment:
There are multiple balance transfer scripts. Can we remove/consolidate some of them? This one seems to be redundant, especially bcoz the only unique part of creating appID is probably commented out. Let us remove this script if it is not showing anything new and instead put this as part of one of the tests inside gsrpc, if possible.